README.md

```diff
@@ -28,6 +28,7 @@ Current status for the https://developer.ibm.com/exchanges/models/ ecosystem.
 | Video    | [Sports Video Classifier](https://github.com/IBM/MAX-Sports-Video-Classifier) | [![Build Status](https://travis-ci.com/IBM/MAX-Sports-Video-Classifier.svg?branch=master)](https://travis-ci.com/IBM/MAX-Sports-Video-Classifier) [![API demo](https://img.shields.io/website/http/MAX-Sports-Video-Classifier.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=API%20demo&down_message=down&up_message=up)](http://MAX-Sports-Video-Classifier.max.us-south.containers.appdomain.cloud/) | [![GitHub release](https://img.shields.io/github/release/IBM/MAX-Sports-Video-Classifier.svg)](https://github.com/IBM/MAX-Sports-Video-Classifier/releases/latest) |
 | NLP | [Review Text Generator](https://github.com/IBM/MAX-Review-Text-Generator) | [![Build Status](https://travis-ci.com/IBM/MAX-Review-Text-Generator.svg?branch=master)](https://travis-ci.com/IBM/MAX-Review-Text-Generator) [![API demo](https://img.shields.io/website/http/MAX-Review-Text-Generator.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=API%20demo&down_message=down&up_message=up)](http://MAX-Review-Text-Generator.max.us-south.containers.appdomain.cloud/) | [![GitHub release](https://img.shields.io/github/release/IBM/MAX-Review-Text-Generator.svg)](https://github.com/IBM/MAX-Review-Text-Generator/releases/latest) |
 |          | [News Text Generator](https://github.com/IBM/MAX-News-Text-Generator) | [![Build Status](https://travis-ci.com/IBM/MAX-News-Text-Generator.svg?branch=master)](https://travis-ci.com/IBM/MAX-News-Text-Generator) [![API demo](https://img.shields.io/website/http/MAX-News-Text-Generator.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=API%20demo&down_message=down&up_message=up)](http://MAX-News-Text-Generator.max.us-south.containers.appdomain.cloud/) | [![GitHub release](https://img.shields.io/github/release/IBM/MAX-News-Text-Generator.svg)](https://github.com/IBM/MAX-News-Text-Generator/releases/latest) |
+|          | [Question Answering](https://github.com/IBM/MAX-News-Question-Answering) | [![Build Status](https://travis-ci.com/IBM/MAX-Question-Answering.svg?branch=master)](https://travis-ci.com/IBM/MAX-Question-Answering) [![API demo](https://img.shields.io/website/http/max-question-answering.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=API%20demo&down_message=down&up_message=up)](http://max-question-answering.max.us-south.containers.appdomain.cloud/) | [![GitHub release](https://img.shields.io/github/release/IBM/MAX-News-Text-Generator.svg)](https://github.com/IBM/MAX-question-answering/releases/latest) |
```

Comment:
```suggestion
|          | [Question Answering](https://github.com/IBM/MAX-Question-Answering) | [![Build Status](https://travis-ci.com/IBM/MAX-Question-Answering.svg?branch=master)](https://travis-ci.com/IBM/MAX-Question-Answering) [![API demo](https://img.shields.io/website/http/max-question-answering.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=API%20demo&down_message=down&up_message=up)](http://max-question-answering.max.us-south.containers.appdomain.cloud/) | [![GitHub release](https://img.shields.io/github/release/IBM/MAX-Question-Answering.svg)](https://github.com/IBM/MAX-question-answering/releases/latest) |
```

Reply:
^ the suggestion didn't format properly. 

`https://github.com/IBM/MAX-News-Question-Answering` -> `https://github.com/IBM/MAX-Question-Answering`
`https://img.shields.io/github/release/IBM/MAX-News-Text-Generator.svg` -> `https://img.shields.io/github/release/IBM/MAX-Question-Answering.svg`